proposals/nomination-schedule.md

```diff
@@ -0,0 +1,24 @@
+# Proposed Nomination Schedule
+
+
+
+| Event                          | Time Taken  | What's involved |
+| :------------------------------ |:-----------| :---------------|
+| Marketing of board nominations | 1 week      | - Raise awareness of the board nominations coming up. <br/> - Share requirements for being eligible for being a board member. <br/> - Share the commitments and responsibilities of being a board member.<br/> - Share how to nominate yourself or someone else for the board. <br/> - Share how and why to become a member of the .NET Foundation.  |
+| Application Period             | 3 - 4 weeks  | - Nomination process is marketed. <br> - People nominate themselves and others for the board positions. <br/> - Nomination committee reaches out to potential candidates to go forward for the board. |
+| Nomination committee shortlist | 1 week      | - Nomination committee produces shortlist from submissions based on set criteria. |
+| Promotion of candidates        | 2 weeks | - Shortlist of candidates is shared with the public. <br/> - Promotional activities to share more about the candidates to help combat the "popularity contest". |
+| Voting period                  | 2 weeks     | - Members to vote on who they want as the board members. |
+
+<br/>
+
+## Lessons from last time
+
+- We need a clear and universal message for what are the requirements for becoming a board member. 
+  - They need to be a .NET Foundation member.
+  - They need to identify with one or more of the member participation types (don't have to be OSS contributors).
+  - How this aligns to the purpose of the .NET Foundation (that the .NET Foundation is not purely for OSS).
+- Promotion and outreach with diverse groups could be beneficial. We struggled with representation last year.
+- Clarity on what is involved in being a board member may help those who aren't sure about going forward for it.
+- A list of members for the nomination committee will help them know who they can reach out to and offer support with their applications.
+- Are there alternate channels we can promote through beyond the .NET Foundation media channels? Such as asking members who run events to promote the board nomination process.
```

Comment:
Other random notes I have from our meeting on lessons learned:

- When reaching out to potential members of the nominating committee, be sure to explain that nominating committee members can't stand for election.
- We need to reach out underrepresented geographies and groups for the nominating committee to encourage members of those groups and geographies to run for board seats.
-